.github/workflows/labeler.yml

```
@@ -1,7 +1,9 @@
 name: "Pull Request Labeler"
 on:
-- pull_request_target
-
+  pull_request:
+    types: [opened]
+    paths:
+      - 'content/**'
```

Regarding https://github.com/cncf/glossary/pull/472#issuecomment-1049525204, https://github.com/cncf/glossary/pull/472#issuecomment-1049543593 and https://github.com/cncf/glossary/pull/473#issuecomment-1049530965:

```suggestion
  pull_request_target:
    types: [opened, synchronize]
    paths:
      - 'content/**'
    branches:
      - '**'
```

@seokho-son PTAL!

---

For anyone's information:
the difference between `pull_request` and `pull_request_target`:

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#:~:text=This%20event%20runs,requests%20from%20forks.
> This event (`pull_request_target`) runs in the context of the base of the pull request, rather than in the context of the merge commit, as the `pull_request` event does. This prevents execution of unsafe code from the head of the pull request that could alter your repository or steal any secrets you use in your workflow. This event allows your workflow to do things like label or comment on pull requests from forks. Avoid using this event if you need to build or run code from the pull request.

so it seems that `pull_request_target` is ok for here. 😊